dynamic_programming/fizz_buzz.py

```diff
@@ -0,0 +1,11 @@
+#https://en.wikipedia.org/wiki/Fizz_buzz#Programming
+number = 1
+out = ""
+iterations = 100
+
+while number <= int(iterations):
```

Comment:
```suggestion
while number <= iterations:
```
`iterations` is already of type `int`